{"name": "test_accept_order_twice_error", "status": "passed", "description": "Проверка невозможности принятия заказа, если его уже взяли в работу. Получаем код 409 и сообщение 'Этот заказ уже в работе'", "steps": [{"name": "Принять заказ и получить его id", "status": "passed", "steps": [{"name": "Создать заказ", "status": "passed", "parameters": [{"name": "payload", "value": "'{\"firstName\": \"\\u041f\\u0440\\u043e\\u043a\\u043b\", \"lastName\": \"\\u0416\\u0443\\u0440\\u0430\\u0432\\u043b\\u0435\\u0432\", \"address\": \"\\u0441\\u0442. \\u0414\\u043e\\u043b\\u0438\\u043d\\u0441\\u043a, \\u0430\\u043b\\u043b. \\u041f\\u043e\\u0433\\u0440\\u0430\\u043d\\u0438\\u0447\\u043d\\u0430\\u044f, \\u0434. 5 \\u043a. 32, 518065\", \"metroStation\": 6, \"phone\": \"8 236 749 7732\", \"rentTime\": 1, \"deliveryDate\": \"2011-01-14\", \"color\": \"BLACK\", \"comment\": \"\\u0422\\u0435\\u0440\\u0430\\u043f\\u0438\\u044f.\"}'"}], "start": 1718029973450, "stop": 1718029973514}, {"name": "Получить заказ по его номеру", "status": "passed", "parameters": [{"name": "track_number", "value": "41112"}], "start": 1718029973514, "stop": 1718029973636}], "start": 1718029973414, "stop": 1718029973636}, {"name": "Создание и Логин курьера", "status": "passed", "parameters": [{"name": "payload", "value": "{'login': 'mrkzlhccvt', 'password': 'hogunxoaav', 'firstName': 'vpkbfduvxw'}"}], "start": 1718029973636, "stop": 1718029973817}, {"name": "Принять заказ", "status": "passed", "parameters": [{"name": "id_order", "value": "300363"}, {"name": "id_courier", "value": "330525"}], "start": 1718029973817, "stop": 1718029973879}, {"name": "Принять заказ", "status": "passed", "parameters": [{"name": "id_order", "value": "300363"}, {"name": "id_courier", "value": "330525"}], "start": 1718029973879, "stop": 1718029973964}], "start": 1718029973414, "stop": 1718029973979, "uuid": "dd8464bd-58b3-4cc2-a38d-9cb3454025cf", "historyId": "25657ea45580c9f35f74fd7fe579c48e", "testCaseId": "25657ea45580c9f35f74fd7fe579c48e", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_twice_error", "labels": [{"name": "feature", "value": "Ручка /api/v1/orders/"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "NB-MP25VCCB"}, {"name": "thread", "value": "21144-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}